src/System.Net.Requests/tests/FileWebRequestTest.cs

```diff
@@ -109,10 +108,8 @@ public abstract class FileWebRequestTestBase
                 using (WebResponse response = await GetResponseAsync(request))
                 {
                     Assert.Equal(data.Length, response.ContentLength);
-                    Assert.Equal(data.Length.ToString(), response.Headers[HttpRequestHeader.ContentLength]);
 
                     Assert.Equal("application/octet-stream", response.ContentType);
-                    Assert.Equal("application/octet-stream", response.Headers[HttpRequestHeader.ContentType]);
```

Comment:
The asserts are invalid.  They throw on Desktop.  See #18784.  Since this is a FileWebRequest, there are no 'Content-Length' nor 'Content-Type' headers.  It is an invalid concept for a WebRequest that is a FileWebRequest.